osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/shared/services/AbstractResponseDataCleanupSchedulingConfig.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.shared.services;
+
+import org.opensmartgridplatform.shared.application.config.AbstractSchedulingConfig;
+import org.opensmartgridplatform.shared.application.config.SchedulingConfigProperties.Builder;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.springframework.beans.factory.annotation.Value;
+
+/**
+ * Base class for ResponseDataCleanupConfig classes for components of OSGP.
+ */
+public abstract class AbstractResponseDataCleanupSchedulingConfig extends AbstractSchedulingConfig {
+
+    @Value("${db.driver}")
+    protected String databaseDriver;
```

Comment:
I will look into this when working on 4697 because there are different approaches used by the classes inheriting from `AbstractSchedulingConfig`, like using `@Value` versus using a `@Bean` function.